tutor/src/models/task-plans/teacher/scores.js

```diff
@@ -244,9 +244,8 @@ class TaskPlanScoresTasking extends BaseModel {
   @identifier id;
   @field period_id;
   @field period_name;
+  @field total_fraction;
 
-  @field({ type: 'object' }) average_score;
-  @field({ type: 'object' }) available_points;
   @belongsTo({ model: 'task-plan/scores' }) plan;
```

Comment:
I dunno why these were here.  the BE doesn't send those fields but caused some confusion since it seemed like they'd do what we wanted